From: Hannes Reinecke <hare@suse.de>
Date: Wed, 28 Sep 2016 15:22:45 +0200
Subject: [PATCH] scsi: Do not use synchronous ALUA submission for NetApp arrays
References: bsc#1001688
Git-commit: 9c58b395563b15866c9fd9bb7e6f7f8db12ac3ae
X-notes: Same patch, different source
Patch-mainline: v4.10-rc1

Both NetApp FAS and NetAPP E-series should be capable of sustaining
asynchronous STPG submissions, so we don't need to blacklist them
with BLIST_SYNC_ALUA.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 drivers/scsi/scsi_devinfo.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/scsi/scsi_devinfo.c b/drivers/scsi/scsi_devinfo.c
index da19a68..cab178f 100644
--- a/drivers/scsi/scsi_devinfo.c
+++ b/drivers/scsi/scsi_devinfo.c
@@ -223,8 +223,6 @@ static struct {
 	{"NAKAMICH", "MJ-5.16S", NULL, BLIST_FORCELUN | BLIST_SINGLELUN},
 	{"NEC", "PD-1 ODX654P", NULL, BLIST_FORCELUN | BLIST_SINGLELUN},
 	{"NEC", "iStorage", NULL, BLIST_REPORTLUN2},
-	{"NETAPP", "LUN C-Mode", NULL, BLIST_SYNC_ALUA},
-	{"NETAPP", "INF-01-00", NULL, BLIST_SYNC_ALUA},
 	{"NRC", "MBR-7", NULL, BLIST_FORCELUN | BLIST_SINGLELUN},
 	{"NRC", "MBR-7.4", NULL, BLIST_FORCELUN | BLIST_SINGLELUN},
 	{"PIONEER", "CD-ROM DRM-600", NULL, BLIST_FORCELUN | BLIST_SINGLELUN},
-- 
1.8.5.6

